airbyte-protocol/protocol-models/src/main/java/io/airbyte/protocol/models/JsonSchemaType.java

```diff
@@ -72,7 +88,11 @@ public static class Builder {
 
     private Builder(final JsonSchemaPrimitive type) {
       typeMapBuilder = ImmutableMap.builder();
-      typeMapBuilder.put(TYPE, type.name().toLowerCase());
+      if (JsonSchemaPrimitiveUtil.isV0Schema(type)) {
+        typeMapBuilder.put(TYPE, type.name().toLowerCase());
+      } else {
+        typeMapBuilder.put(REF, PRIMITIVE_TO_REFERENCE_BIMAP.get(type));
```

Comment:
Should there be a `toLowercase()`?

Reply:
No, the string that `PRIMITIVE_TO_REFERENCE_MAP` returns is already lowercase